src/setup/sync.ts

```diff
@@ -1,7 +1,7 @@
 import { DevSecurityLevel, Shardus, ShardusTypes } from '@shardus/core'
 import { Account, Address } from '@ethereumjs/util'
 import config from '../config'
-import genesis from '../config/genesis.json'
+import genesis from '../config/genesis_accounts.json'
```

Comment:
Let's refrain from renaming any files at this point where it isn't strictly necessary.

Reply:
I was instructed by the team to rename the file but it is alright. Please inform me what the final changes needed are, if any.